# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/HttpTransactionChild.cpp
# Commit: 96bd98319c62
# Full Hash: 96bd98319c6242a9b05dc4723b928e88537424e1
# Author: Kershaw Chang <kershaw@mozilla.com>
# Date: 2020-08-26 03:24:37
# Regressor Bug: 1652667
# File Overlap Count: 1
# Description:
#   Bug 1652667 - P1: Add telemetry for when an HTTPSSVC record is received r=dragana
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D87844
# ==============================================================================

diff -r b61a789151df -r 96bd98319c62 netwerk/protocol/http/HttpTransactionChild.cpp
--- a/netwerk/protocol/http/HttpTransactionChild.cpp	Tue Aug 25 17:29:21 2020 +0000
+++ b/netwerk/protocol/http/HttpTransactionChild.cpp	Tue Aug 25 17:29:32 2020 +0000
@@ -492,12 +492,14 @@
   int32_t proxyConnectResponseCode =
       mTransaction->GetProxyConnectResponseCode();
 
+  Maybe<uint32_t> stage = mTransaction->HTTPSSVCReceivedStage();
+
   Unused << SendOnStartRequest(status, optionalHead, serializedSecurityInfoOut,
                                mTransaction->ProxyConnectFailed(),
                                ToTimingStructArgs(mTransaction->Timings()),
                                proxyConnectResponseCode, dataForSniffer,
                                optionalAltSvcUsed, !!mDataBridgeParent,
-                               mTransaction->TakeRestartedState());
+                               mTransaction->TakeRestartedState(), stage);
   return NS_OK;
 }
 